x/data/features/msg_define_resolver.feature

```diff
@@ -0,0 +1,23 @@
+Feature: MsgDefineResolver
```

Comment:
can rename the directory to `acceptance-tests`?

Reply:
Using features for the directory name is actually a standard practice when writing features. Check out the examples in gocuke or the cucumber documentation if you're interested in learning more:

- https://github.com/regen-network/gocuke
- https://cucumber.io/docs/gherkin/reference/

Reply:
I wouldn't call it it a _standard practice_. The file is _feature_ file of an acceptance test. Usually we define test in spec or test directories.

Reply:
I think it's safe to call it a standard practice and have not seen an example that does not use the `features` as the name of the directory. Seems like a silly thing to debate over.

Reply:
Seems like a silly thing to debate over. I think it's safe to call it a standard practice and have not seen an example that does not use `features` as the name of the directory.

Reply:
I would recommend checking out some of the resources. I think standard practice is pretty accurate. We define golang tests in a test directory but features are typically stored in a features directory.

Reply:
I prefer features/ personally